recipes/simde/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "simde" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simd-everywhere/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 366d5e9a342c30f1e40d1234656fb49af5ee35590aaf53b3c79b2afb906ed4c8
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
```

Comment:
Why do we need a `run_exports` if this package consists only of header files?

Reply:
Done.